_projects/fmm_project.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: page_project
+title: Scalability Enhancements to FMM for Molecular Dynamics Simulations
+subtitle:
+date: 01/01/2016
+update:
+navbar: Projects
+subnavbar:
+footer: true
+logo:
+project_url:
+short_desc:
+---
+<!---
+Questions/requests:
+
+* we need a naming convention for the project_template...
+
+* what is "update" and "subnavbar"?
+
+* why do we need a date? If it's just the date of creation/change, this should
+not be changed by the author of this page. also: how do I know what format the
+date is required?
+
+* we need meta-data here, in particular the particiants/members of the project
+
+* how do I link to the people in the directory when I write text? The same will
+be important for affiliations and references.
+
+* probably remove "short_desc" in favor of other data like members and/or
+affiliations
```

Comment:
We may remove this anyway.. the cards could also consist of title + people + affiliations only. I don't want to force people to write new text..